conf/modules/dysgu.config

```
@@ -0,0 +1,28 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+        ext.when   = When to run the module.
+----------------------------------------------------------------------------------------
+*/
+
+// DYSGU
+
+process {
+    if (params.tools && params.tools.split(',').contains('dysgu')) {
+        withName: 'DYSGU_GERMLINE' {
```

```suggestion
        withName: 'DYSGU' {
```

I think the process is just `DYSGU`

---

Hi @maxulysse i accepted your commits could you please review the changes and let me know if it's ready to merge?